### ISSUE TITLE
Panels have stopped working

### ISSUE THREAD

**Issue by johnksawers** [mbnuqw/sidebery#352]
## Steps to reproduce
First of all, thanks for making this, it's among my best plugins you can get for Firefox. It was shortly after I added a new container and panel (7 in total), and I think it was right around the FF 86.0 update, but I can't remember if it was that update the did it. But No panels will display now, they show in the drop-down, but when selected only the 'new tab' page appears. I've tried clearing storage (though there was nothing stored), resetting settings, reloading Sidbery, deleting all panels and re-creating. 

## Expected behavior
Panels show in the panel and allow be to switch between them by clicking the panel icon

## Actual behavior
Panels won't display in the panel bar, they only appear int the drop-down. Bookmarks and settings are the only things there. When selecting a panel from the list, the new tab page appears but the panel doesn't activate. 

Happy to dig up debug logs or anything else to help you track this down. 

## Environment info

OS:  
Firefox version:  
Addon version:  

<details><code>

</code></details>

**Comment by jlokier** [comment]
Are you sure it's the panel not being displayed?  It may be that the tabs in those panels are gone, and it's hiding empty panels from the switcher.

I'm seeing a similar symptom now after starting up Firefox today - all panels except the first are hidden in the switcher, but available in the drop-down.

When I switch to the other panels and get only New Tab, I think it's showing the panel but all of my tabs in the non-Default panels have been lost.  Looking at snapshots made today, they only list Default panel entries.

**Comment by jlokier** [comment]
Or maybe what I'm seeing is the tabs all moving to the default panel, as described in https://github.com/mbnuqw/sidebery/issues/267#issuecomment-643784615

It's hard to be sure.  When Firefox first started up, all the windows were empty with no tabs, yet restarting Firefox brought them all back. Strange session behaviour indeed.

**Comment by jlokier** [comment]
No, after comparing with current contents, it looks more like all the tabs in my non-Default panels have disappeared, rather than been moved to the Default panel.

**Comment by sylvoie** [comment]
Hello, 

I get the same behavior from sidebrry, I mean, all tabs move to my default panel. Others non-default panels emptied from their content.

Thank's to correct that situation, it's annoying to move tabs from default panel to others panels!
